### DISCUSSION TITLE
Dust discrepancy between BLT1850 and FMTHIST

### DISCUSSION THREAD

**Discussion by cecilehannay** [NCAR/cesm_dev#30] [2. Model bias]
We noticed that since we used the new dust scheme, there are large differences between **pre-industrial B cases** and **present-day F cases**. 
For similar dust parameters,
```
AODDUST = 0.0287 in FMTHIST
AODDUST = 0.0174 in BLT1850
```

The two cases use different compsets and different time periods. 

One other distinction is that the F-case use `CLM%SP` and the B-case uses `CLM%BGC`, which could lead to differences in LAI. And even if they were both BGC, there would be different land properties in these two different time periods.

Both runs use:
```
dust_emis_method = 'Leung_2023'
dust_emis_fact = 2.3
```

Lands diagnostics are [here](https://webext.cgd.ucar.edu/BLT1850/b.e30_beta04.BLT1850.ne30_t232_wgx3.116/lnd/b.e30_beta04.BLT1850.ne30_t232_wgx3.116_14_23-f.e30_alpha04a.FMTHIST.ne30_L93_tuning.008_2000_2009/setsIndex.html)

The runs are in (until there are scrubbed)
`/glade/derecho/scratch/hannay/archive/b.e30_beta04.BLT1850.ne30_t232_wgx3.116`
and 
`/glade/derecho/scratch/hannay/archive/f.e30_alpha04a.FMTHIST.ne30_L93_tuning.008`

Maybe everything is ok and the differences are expected but we want someone with expertise to look at this. 

See also: https://github.com/NCAR/cesm_dev/issues/19#issuecomment-2480053093

@dmleung, @PeterHjortLauritzen, @adamrher, @dlawrenncar, @tilmes, @olyson, @slevis-lmwg, @wwieder

**Comment by cecilehannay** [comment]
From I understood from previous email exchanges: 

- The lower AODDUST in the B case aligns better with historical sedimentary records, showing ~50% less AODDUST in 1850 compared to the 2000s
- The dust scheme in Leung2023 appears more sensitive to land variables (e.g., TLAI, FV, H2OSOI) than Zender2003, though both consider these factors.
- Initial plots show that TLAI and dust emissions behave as expected, suggesting that the dust response in the B case is reasonable given its land conditions.

<img width="1063" alt="image001 (1)" src="https://github.com/user-attachments/assets/16925304-45b9-4ea4-84f3-e47bc8255cf7" />

@dmleung: correct me if I am wrong? Please could you confirm there is nothing to worry about.

**Reply by wwieder** [reply]
Thanks for starting this discussion @cecilehannay.  I also wondered if @dmleung can confirm the dust model uses the prognostic LAI from a BGC run, or does it use a SP LAI under the hood (even in BGC cases)?

I'm also curious what soil moisture information (variable name and layer) are used for the dust scheme, as SOILLIQ is higher in the B case at the 4th soil layer (which I would assume would keep the dust down the in B-case), but differences are rather muted in the top soil layer (second plot).   
![image](https://github.com/user-attachments/assets/bc66b38b-5807-4b7c-84af-fee6a0fd7053)

![image](https://github.com/user-attachments/assets/6361acbc-1b1d-4643-b531-4d4a4471bc9f)

Finally, winds look lower over some arid regions in the b-case, which could help explain the AODDUST results?
![image](https://github.com/user-attachments/assets/85eed5ae-86fc-4f2c-b009-ba1d0d40995d)

**Reply by wwieder** [reply]
I'll retract my _wetter soils_ hypothesis, as it looks like the dust scheme is just looking at the [top soil layer](https://github.com/ESCOMP/CTSM/blob/1d824b8c2eb95159d8e247ca7c71cd177223de59/src/biogeochem/DustEmisLeung2023.F90#L588).

**Reply by wwieder** [reply]
regarding my LAI question, I can't really tell what the dust and MEGAN schemes are doing in [clm_Driver](https://github.com/ESCOMP/CTSM/blob/1d824b8c2eb95159d8e247ca7c71cd177223de59/src/main/clm_driver.F90#L802)?  Maybe @ekluzek can clarify?

**Reply by dmleung** [reply]
@cecilehannay: 
- I think what you said is correct. 
- From the plot Cecile showed it looked like the spatial distribution of friction velocity for dust emission (WND_FRC_SOIL; upper row) in BLT1850 followed the spatial LAI distribution. The lower row shows global LAI in the 1850s, with colorbar saturated at 1. LAI values in the 1850s were higher than the right column over places like N America, NW China, the Sahel, the Horn of Africa, and Australia. So, it looks like to me that WND_FRC_SOIL and dust emissions in Leung_2023 is behaving normally following the changes in land conditions (LAI in particular).
- The question to me is more about if the LAI distribution in the BLT1850 is correct. @wwieder or @lawrencepj1 do you think LAI looked right in the 1850s in a BGC-CROP case? I can replot it if you don't want the colorbar to saturate at 1.
- I tend to believe that BLT1850's being able to reduce dust by a good amount is a coincidence with what the core records show. We want dust to be 50 % in the 1850s lower than the 2000s, because this is what the core records show. We showed that in the FHIST case we could not show this historical dust trend. But for now I am not sure if this higher LAI from a BGC-CROP case is the solution.

**Reply by dmleung** [reply]
@wwieder:
- For the BLT850 case the compset is 1850_CAM70%LT_CLM60%BGC-CROP... and Component LND is clm6.0:BGC (vert. resol. CN and methane) with prognostic crop. Is there a namelist variable to check whether SP LAI data is used?
- For the FLTHIST, LND is clm6.0:Satellite phenology.
Case directory (from Adam):
BLT1850: /glade/campaign/cesm/cesmdata/cseg/runs/cesm2_0/b.e30_beta04.BLT1850.ne30_t232_wgx3.116
FMTHIST: /glade/campaign/cesm/cesmdata/cseg/runs/cesm2_0/f.e30_alpha04a.FMTHIST.ne30_L93_tuning.008

- I also think wind speed could partially explain some reductions in dust emissions in the BLT1850 case, given wind is smaller over the Sahara, the Horn of Africa, and NW China than the right column. Thanks Will for plotting the winds out.
- Yes, dust emission schemes (Zender_2003 and Leung_2023) only see the topmost soil layer. The deeper layers and the dry surface layer (DSL) are not considered. 
- I usually look at H2OSOI (m3/m3) and GWC (kg/kg) at the topmost soil layer. Good to know about SOILLIQ (kg/m2) too.

**Reply by ekluzek** [reply]
@wwieder dust is using the standard tlai and tsai from CanopyState. So it's only MODIS LAI for SP compsets, BGC will be used the BGC predicted LAI+SAI.

You are right that MEGAN does the bizarre thing to use LAI from the surface dataset. But, DUST does NOT.

Is that all you need from me here?

**Reply by wwieder** [reply]
Yes, thanks Erik.

**Comment by dlawrenncar** [comment]
If the main question is whether or not the dust emissions in B1850 are
reasonable and that the difference between B1850 and F2000 seems
reasonable, we could do two tests.  First, would be a F1850 case.  Second
would be to look at the trends in dust emissions in a land-only historical
case.  The difference in dust AOD between B1850 and F2000 does seem pretty
large and while the historical record may indicate that dust emissions
between 1850 and 2000 should be different, my understanding was that CESM
was not able to reproduce that trend, even with the new dust scheme and
that this was an area of active research (e.g., land degradation, which is
not captured in the land model (think dust bowl), is one candidate reason
why the model cannot reproduce observed dust trends, but there are
others).

**Reply by dmleung** [reply]
- I agree that a test with F1850 will make things clear -- a case in the 1850s with SP LAI instead of BGC-CROP. From my previous 1851–2000 run with FHIST in CESM2.2 (in an AeroCom experiment) I think both LAI and dust emissions will look like those in FMT2000.
- A case with IHIST will be interesting too, although for now I suppose the model does not do dust trend right (as seen in my previous 1851–2000 run with FHIST).

**Reply by wwieder** [reply]
One point of clarification, I thought FHIST cases were only 1979-2015 because they use observed SSTs and F1850 wasn't really a thing?

Regardless, in I cases we see a decline in dust fluxes using Leung #69, below.  whereas with the Zender scheme (#67) emissions were flat.  [Diagnostics here](https://webext.cgd.ucar.edu/I20TR/ctsm53n04ctsm52028_f09_hist/lnd/ctsm53n04ctsm52028_f09_hist_1995_2014-ctsm51_ctsm52d004_ne30pg3t232_GSWP3V1_ABsnoCDE_blk_A5BCD_hist_1995_2014/setsIndex.html)

![image](https://github.com/user-attachments/assets/c680b078-0a82-4bc8-bcf5-49a76f7a678a)

To your question on diagnostics, we could do a B1850-SP case?  Alternatively, we can look at diagnostics from an existing FLT2000-BGC case, [maybe this one from LMWG_dev](https://github.com/NCAR/LMWG_dev/issues/78) vs a B1850-BGC case?  @dmleung would it be helpful to compare different time slices from two BGC runs?

**Reply by dmleung** [reply]
@wweider 
- Interesting, I have never run an I case for 1850–2000. If we look at global mean TLAI trend which is increasing, it also explains the dust emissions reductions in my case (although this LAI time series is of course a global mean). Again, it looks to me Zender_2003 is not very sensitive to input met fields and lnd fields.
- I am a little curious about the (GSWP) wind trend as well, but it looks like there are no wind outputs in the land-only case.
- B1850-SP case helps too, just like F1850-SP, as long as we are using SP LAI in the 1850s.
- I don't know, it depends on how the BGC LAI in the 2000s looks like the SP LAI in the 2000s. I have no idea how BGC LAI looks like in 1850 and 2000, but when I saw the BGC-CROP LAI in the 1850s in Cecile's plot, I was quite certain that dust emissions changes were mainly due to the fact that BGC-CROP LAI in the 1850s look different from the SP LAI in the 2000s.

<img width="565" alt="image" src="https://github.com/user-attachments/assets/d7194cdb-61d6-4ed5-a4ef-389f7e05f9a6" />

**Reply by dmleung** [reply]
Soil moisture (SOILLIQ) also increased throughout the historical period (in the I case). This also can contribute to my scheme's dust reductions. Again, maybe Zender's scheme is not responding much to the changes in lnd fields.

<img width="1134" alt="image" src="https://github.com/user-attachments/assets/222dcba5-4d7e-4e38-a550-bff789d91ce1" />

**Reply by dmleung** [reply]
@wwieder To my understanding in an F case
SSTICE_DATA_FILENAME="$DIN_LOC_ROOT/atm/cam/sst/sst_HadOIBl_bc_1x1_1850_2021_c120422.nc"
so they have reanalysis data back to the 1870s. I think 1850–1870 uses the SST data for 1870-1890, if I remember correctly. @cecilehannay can clarify if I was wrong.

**Comment by wwieder** [comment]
One more plot on this topic.  [B-104](https://github.com/NCAR/amwg_dev/issues/575) was an old historical run that used the Zender scheme, which looks like it has flat dust emissions throughout the historical run.  Since this is similar to what we saw with Zender in the IHIST case (above) I might not expect increases in AODDUST over the historical period with Leung.  @cecilehannay was your concern with the total AODDUST burden over the observational record, or more of a curriosity with the difference between the B1850-BGC and F2000-SP cases?  

![image](https://github.com/user-attachments/assets/fb92650d-d44f-497f-acc8-0e19d42ed7b5)

**Comment by dlawrenncar** [comment]
I'll also note that looking at some maps between B1850 cases and F2000 cases (with CLMSP), I see differences in Dust emissions and LAI between the two runs that I think would explain the dust differences.  The higher emissions in F2000(CLMSP) cases are associated with lower LAI over Australia and southern S. America and Sahel.
![image](https://github.com/user-attachments/assets/d7556e5f-a2d1-4f55-8835-faccaa9aab3a)
![image](https://github.com/user-attachments/assets/56ad7c92-6e06-4f32-b057-34ae0927fc87)

So, my guess is this is a LAI bias (or at least an LAI difference problem).  The CLM team will be working to improve LAI through the calibration process.  In the context of the other issues that we are having (e.g., with RESTOM), we may want to keep in mind that dust emissions will be strongly controlled by the LAI, especially in these semi-arid regions.

**Comment by dmleung** [comment]
Hi @cecilehannay , do you have a BLTHIST or BMTHIST case for beta04 or 05 using Leung_2023 dust that is run from 1850 all the way to the 2000s or 2010s? I wanted to take a look and see if dust would behave normally in the contemporary climate in a B case (assuming CLM60%BGC-CROP being used). Let me know... Thanks!

**Reply by wwieder** [reply]
Thanks @dmleung the latest BHIST case is 121 (issue #34). 

NOTE: There are a number of climate (and LAI) biases in this run, but hopefully we'll get another historical run before too long...

**Reply by dmleung** [reply]
Thanks @wwieder for the information, and also for reminding me to look at dust from time to time. :)
The BLTHIST 121 case from issue #34 was run from 1850 to 2012.
/glade/campaign/cesm/development/cross-wg/cesm3_development/b.e30_beta04.BLTHIST.ne30_t232_wgx3.121/
LAI in the 2000-2009 was very similar to the first plot above, so the BGC-CROP LAI in the B case is still higher than the SP LAI in the F case over arid regions. Global mean AODDUST = 0.179 in the 2000s, similar to the 0.174 above for the 116 case. 
Let me know once the LAI bias is fixed and I will look at dust in a later case again. Thanks!
